src/awkward/operations/ak_corr.py

```diff
@@ -150,5 +152,4 @@ def _impl(x, y, weight, axis, keepdims, mask_identity):
                 highlevel=True,
                 behavior=behavior,
             )
-        nplike = ak._nplikes.nplike_of(sumwxy, sumwxx, sumwyy)
-        return nplike.true_divide(sumwxy, nplike.sqrt(sumwxx * sumwyy))
```

Comment:
@jpivarski I dropped the use of the nplike here, because we're operating on high level `ak.Array`. In this PR, it's being made more explicit that nplikes only operate upon their low level arrays.

However, the problem is that we need to invoke a ufunc via Awkward's ufunc machinery. Elsewhere, we also need the `np.ufunc` symbols in the modules that register awkward behaviours.

A forthcoming PR will replace the `NumpyMetadata` class with a `metadata` module, but that's for dtypes and constants rather than ufuncs. 

I was originally thinking that we wanted to drop the use of `import numpy` within Awkward so that we can be confident no code is sneakily using it by accident.  So, my solution here is to explicitly import numpy as `_numpy_ufuncs`, so that we could hypothetically implement a plugin to only allow imports of numpy that reflect it's intention to be used for e.g ufuncs only. It seems like a least-worst option.